the-collab-lab/tcl-21-smart-shopping-list
# As tm navigate lists

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

1. Added react-router-dom has been added as a project dependency

2. Links are present and persistent at the bottom of the app: one for the “list” view, the other for the “add an item” view

3. When one of the links is clicked, the browser URL updates to represent the current view and a matching view component is displayed

4. Whichever view is selected, the corresponding link should display in bold text



## Related Issue

closes 2



## Acceptance Criteria

- The following has been added as a project dependency: react-router-dom
- Added links to a "List" view and "Add An Item" view
- The browser URL will correspond with the changing views via the clicked links



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After




## Testing Steps / QA Criteria

1. Pull down remote branches: git pull
2. Navigate to correct branch: git checkout as-tm-navigate-lists
3. Install react-router-dom as dependency using npm install
4. Run server and view in browser on localhost:3000: npm start
5. Click links "List" and "Add an Item" to navigate to their respective pages
6. The active ('clicked') link should in bold
